stores/preferences.ts

```diff
@@ -268,6 +268,8 @@ export const usePreferencesStore = defineStore('preferences', {
     },
     setUserLocale(locale: string) {
       const { $i18n } = useNuxtApp()
+      // only supports en for now
+      locale = 'en'
       $i18n.locale.value = locale
       this.userLocale = locale
```

Comment:
oh, I missed this one. based on sonar, should we set like this?
```suggestion
      $i18n.locale.value = 'en'
      this.userLocale = 'en'
```